.github/workflows/verify_consumer_pacts.yml

```diff
@@ -0,0 +1,240 @@
+          # This condition is true when the workflow is triggered by a Pact Broker webhook event.
+          if [[ -n "${{ env.PROVIDER_TAG }}" ]]; then
+            echo "git checkout tags/${{ env.PROVIDER_TAG }} || git checkout ${{ env.PROVIDER_BRANCH }}"
+            git checkout tags/${{ env.PROVIDER_TAG }} || git checkout ${{ env.PROVIDER_BRANCH }}
+
+          # Otherwise, switch to CURRENT_BRANCH if the workflow has been triggered by a
+          # PR commit or merge onto the main branch.
+          elif [[ "${{ github.event_name }}" == "pull_request" ]] || [[ "${{ github.event_name }}" == "push" ]]; then
+            echo "git checkout ${{ env.CURRENT_BRANCH }}"
+            git checkout ${{ env.CURRENT_BRANCH }}
+          fi
+
+          # Echo the HEAD commit of the provider branch that has been switched to.
+          echo "git rev-parse HEAD"
+          git rev-parse HEAD
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Configure pact
+        env:
+          PACTBROKER_APPLICATION_YAML_B64: ${{ secrets.PACTBROKER_APPLICATION_YAML_B64 }}
+        run: |
+          mkdir -p ./service/src/test/resources
+          echo $PACTBROKER_APPLICATION_YAML_B64 | base64 --decode >> ./service/src/test/resources/application.yml
+          PACT_PROVIDER_BRANCH=$GITHUB_REF_NAME
+          echo "PACT_PROVIDER_BRANCH=${PACT_PROVIDER_BRANCH}" >> $GITHUB_ENV
+
+      - name: Verify consumer pacts and publish verification status to Pact Broker
+        id: verification-test
+        env:
+          PACT_BROKER_URL: pact-broker.dsp-eng-tools.broadinstitute.org
+          PACT_PROVIDER_VERSION: ${{ env.PROVIDER_VERSION }}
+          PACT_PROVIDER_BRANCH: ${{ env.PROVIDER_BRANCH }}
+          PACT_PUBLISH_RESULTS: true
+          PACT_BROKER_USERNAME: ${{ secrets.PACT_BROKER_USERNAME }}
+          PACT_BROKER_PASSWORD: ${{ secrets.PACT_BROKER_PASSWORD }}
+        run: |
+          echo "provider-version=${{ env.PACT_PROVIDER_VERSION }}" >> $GITHUB_OUTPUT
+          echo "env.CONSUMER_NAME=${{ env.CONSUMER_NAME }}       # This reflects the consumer name for pact verification (generated by Pact Broker)"
+          echo "env.CONSUMER_BRANCH=${{ env.CONSUMER_BRANCH }}   # This reflects the consumer branch for pact verification (generated by Pact Broker)"
+          echo "env.PROVIDER_BRANCH=${{ env.PROVIDER_BRANCH }}   # This reflects the provider branch to switch to for pact verification"
+          echo "env.CONSUMER_VERSION=${{ env.CONSUMER_VERSION }} # This reflects the consumer version for pact verification (generated by Pact Broker)"
+          echo "env.PROVIDER_VERSION=${{ env.PROVIDER_VERSION }} # Deprecate env.PACT_PROVIDER_COMMIT. This new envvar is used for migrating GIT hash to app versioning"
+          ./gradlew --build-cache verifyPacts
+
+## Temporarily disable          
```

Comment:
what needs to happen for us to enable this?

Reply:
We need to merge the Cromwell PR that sets up the other half of the contract so that the broker knows about everything - should happen very soon.